cmd/prometheus/main.go

```diff
@@ -185,6 +186,18 @@ func (c *flagConfig) setFeatureListOptions(logger log.Logger) error {
 			case "agent":
 				agentMode = true
 				level.Info(logger).Log("msg", "Experimental agent mode enabled.")
+			case "timezones":
+				if runtime.GOOS == "windows" {
```

Comment:
Parsing feature flags should never fail, and I don't think we should have this check. I think we should rely on Go's default behaviour and err in LoadLocation (I guess that's what it does by default).